person.rb

```diff
@@ -24,4 +25,8 @@ def correct_name
   def of_age?
     @age >= 18
   end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] The method for adding rentals should look a little different than this, please have a look at[ this example](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) in a previous lesson